src/zscanner/scanner.js

```diff
@@ -136,15 +137,11 @@ const executeScan = function () {
                 } else {
                     scan_status = 'passed';
                 }
-                if (outputFormat.startsWith("sarif") || outputFormat.endsWith("sarif") ||
-                    outputFormat.startsWith("github_sarif") || outputFormat.endsWith("github_sarif")) {
-                    core.setOutput('sarif_file_path', process.cwd() + '/result.sarif')
-                }
-                core.setOutput('scan_status',scan_status);
-                if (outputFormat.startsWith("json")) {
-                    const output = JSON.parse(stdout);
-                    resolve(output);
+                sarifPath = process.cwd() + '/result.sarif';
```

Comment:
The sarif file has to be generated only based on the user input.

Reply:
yes jeevan, the scan command will generate the sarif based on user input,i.e based on outputformat user selects.
So here we are just checking if the file exists and saving it to output variable